src/main/scala/com/danielasfregola/zucchini/Dictionary.scala

```diff
@@ -16,6 +18,10 @@ object Dictionary extends App {
     texts.flatMap(line => line.split("\\W+"))
   }
 
-  println(extractUniqueWordsFromTextFile("BaconipSum.txt").mkString(","))
+  //  println(extractUniqueWordsFromTextFile("BaconipSum.txt").mkString(","))
 
+  val words: Set[String] = extractUniqueWordsFromTextFile("BaconipSum.txt")
+  val word: Arbitrary[String] = Arbitrary {
```

Comment:
Well done! You can also just write `Arbitrary(Gen.oneOf(words))` - it's the same, just more compact!